### DISCUSSION TITLE
Arduino IDE 2.3.2Problem compiling tdslite 6.0

### DISCUSSION THREAD

**Discussion by wanglegang** [tdslite/tdslite#52] [General]
Arduino IDE 2.2.1 on win11 and tdslite 0.6.0  compiled fine
Arduino IDE 2.2.1 The above versions, such as 2.3.2, report an error
![bc](https://github.com/tdslite/tdslite/assets/16290836/54dd1469-a40c-4c63-a040-bfbf6d18993b)

**Comment by xmkg** [comment]
Hi @wanglegang, thanks for reporting this. I'll check it and get back to you.

**Reply by wanglegang** [reply]
@mustafakemalgilor  Thank you for your reply

**Reply by xmkg** [reply]
Hello,

I've pushed a new release (0.8.0), which should fix the issue. Please note that it may take a while for it to appear in the Arduino Library Manager.

**Reply by wanglegang** [reply]
In the Arduino Library Manager, It has been updated, Thank you very much for everything you have done. This library is very useful. Thank you again!
![8](https://github.com/tdslite/tdslite/assets/16290836/73fd354f-8580-4679-9a76-61ccdb66acd0)

**Reply by xmkg** [reply]
I'm glad to see it worked for you. Please feel free to reach out again if anything else arises.

**Reply by wanglegang** [reply]
Ok, thank you
